app/controllers/SelectedTransactionsController.scala

```diff
@@ -112,8 +116,18 @@ class SelectedTransactionsController @Inject()(resultView: selected_transactions
       case (Some(cashAcc), Some(dates)) =>
 
         apiConnector.postCashAccountStatementRequest(request.eori, cashAcc.number, dates.start, dates.end).map {
+
           case Right(_) => Redirect(routes.ConfirmationPageController.onPageLoad())
 
+          case Left(EntryAlreadyExists) =>
+
+            val startDate = dateAsMonthAndYear(dates.start)
```

Comment:
Better to refactor as below to make it readable (first one  - break after duplicateDates( and keep all the parameters on single line)

            Redirect(routes.SelectedTransactionsController.duplicateDates(
              "cf.cash-account.duplicate.message", dateAsMonthAndYear(dates.start), dateAsMonthAndYear(dates.end)))

or 
Redirect(routes.SelectedTransactionsController.duplicateDates("cf.cash-account.duplicate.message",
              dateAsMonthAndYear(dates.start),
              dateAsMonthAndYear(dates.end)))